src/nett/body/__init__.py

```diff
@@ -4,16 +4,17 @@
 .. module:: nett.body
   :synopsis: Body module for nett
 """
+import ast
+from pathlib import Path
+
+from nett.body import wrappers
+
 
 # all available body types
 types = ["basic", "two-eyed", "ragdoll"]
 
 # ASCII art
 ascii_basic = """
-          ---------------::::::::::::::::::::::::::::::::::::::::::::::::::---------------
```

Comment:
We're cutting off the top of its head?? :(

Reply:
Hahaha, just giving the forehead a 'lil trim